tools/lib/ignoredPaths.js

```diff
@@ -3,6 +3,8 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 module.exports = [
+  'pre-comit',
```

Comment:
Pre-commit is a hook which will run some X script (in our case linter) when you try to do `git commit`.